docs/source/contributing/review_pr_pymc_examples.md

```diff
@@ -0,0 +1,113 @@
+(reveiw_pr_pymc_examples)=
+# Review a PR on pymc-examples
+
+The target audience for this page are reviewers of PRs in pymc-examples
+repo. It mostly gathers resources from its
+[contributing guide](https://github.com/pymc-devs/pymc-examples/blob/main/CONTRIBUTING.md),
+[PR template](https://github.com/pymc-devs/pymc-examples/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
+and the {ref}`jupyter_style` page to
+centralize all resources for reviewers and put special emphasis on
+reviewer tasks and responsibilities.
+
+:::{important}
+The most important guideline is the following: **When you aren't _completely_
+sure about something say so, ask around and (re)read the documentation**.
+:::
+
+pymc-examples is a huge collection of notebooks, covering many fields
+and applications of PyMC and Bayesian statistics. In addition,
+its HTML generation infrastructure has been improved by specialized
+people who were also paid so they could dedicate themselves to this task.
+It is perfectly fine to not know about the topic, not know about
+the tech stack being used or to not be sure about either.
+You can still review, either focus on your expertise or take some
+time going through all the available resources to learn about the parts
+you are not sure about.
```

Comment:
Can also just check formatting and best practices

Reply:
I consider knowing about the formatting, how to add alt text to image in sphinx... to be one type of expertise